docs/guide/cookbook/module.md

```diff
@@ -831,3 +831,33 @@ Inside `check` we tell the filter to just be applied if the attribute is named e
 Inside `filter` we extend the Elasticsearch query-chain by our desired filters, using the `bodybuilder` library syntax.
 
 That's it, now we are able to filter by a complex query in only one line inside VSF.
+
+## 7. Working with translations in module
+
+Translations are provided in `core/i18n/resource/i18n/en-US.csv` file and can be extended / overridden in `src/modules/{my-module}/resource/i18n/en-US.csv` accordingly.
+
+### Example
+
+Here's an example of `en-US.csv` for `en-US` locale:
+
+```csv
+"customMessage","Here is the core message. that can be overwritten in the module"
+```
+
+When you create the `en-US.csv` file within your `src/modules/{my-module}/resource/i18n/` folder and override some messages like:
+
+```csv
+"customMessage","You can define or override translation messages here."
+```
+
+... you may expect that `$t('customMessage)` will return `You can define or override translation messages here.` instead of `Here is the core message. that can be overwritten in the module`.
```

Comment:
`$t('customMessage)` - no ending `'`

Reply:
Okay so I can override Core's translation in the module. What if I have override in both module and theme, what has bigger priority? Please describe it